src/Integration/Binding/ProjectBinderFactory.cs

```diff
@@ -47,21 +47,7 @@ internal ProjectBinderFactory(IServiceProvider serviceProvider, ILogger logger,
 
         public IProjectBinder Get(Project project)
         {
-            var languages = projectToLanguageMapper.GetAllBindingLanguagesForProject(project).ToList();
-            var isCSharpVBLanguage = languages.Contains(Core.Language.VBNET) || languages.Contains(Core.Language.CSharp);
-
-            if (isCSharpVBLanguage)
-            {
-                return new CSharpVBProjectBinder(serviceProvider, projectToLanguageMapper, fileSystem, logger);
-            }
-
-            var isCFamilyLanguage = languages.Contains(Core.Language.C) || languages.Contains(Core.Language.Cpp);
-
-            if (isCFamilyLanguage)
-            {
-                return new CFamilyProjectBinder(projectToLanguageMapper, logger, fileSystem);
-            }
-
```

Comment:
Removing this code caused a number of new code smells relating to unused variables.
I'm going to delete the entire class, so the code smells will go at that point.